src/pages/staking-hub/wrapper.tsx

```
@@ -366,44 +405,129 @@ function WrapperPage() {
+        });
+        const payload = {
+          data: TXdata,
+          signer,
+          safeAddress: selectedAddress as `0x${string}`,
+          smartContractAddress: xHOPR_TOKEN_SMART_CONTRACT_ADDRESS,
+        };
+        await dispatch(
+          signOnly ?
+            safeActionsAsync.createSafeContractTransactionThunk(payload) :
+            safeActionsAsync.createAndExecuteSafeContractTransactionThunk(payload)
+        ).unwrap();
```

ESLint

[error] Missing trailing comma.

(comma-dangle)